components/raftstore/src/store/entry_storage.rs

```diff
@@ -1158,18 +1137,15 @@ impl<EK: KvEngine, ER: RaftEngine> EntryStorage<EK, ER> {
     }
 
     /// Warm up entry cache if the result is valid.
-    ///
-    /// Return true when the warmup operation succeed within the timeout.
-    pub fn maybe_warm_up_entry_cache(&mut self, res: RaftlogFetchResult) -> bool {
-        let low = res.low;
+    pub fn on_async_warm_up_entry_cache_fetched(
```

Comment:
How about just using `on_warm_up_entry_cache_fetched` rather than `on_async_xxx` ? It gives me the illusion that this function is an asynchronous function.